apps/storybook/.storybook/main.js

```diff
@@ -1,6 +1,9 @@
 const path = require('path');
 
 module.exports = {
+  features: {
+    storyStoreV7: true, // https://storybook.js.org/blog/storybook-on-demand-architecture/
+  },
```

Comment:
Storybook v6.4 brings an experimental performance re-architecture that will be officially released in v7. Stories are loaded lazily, which means the main bundle is much smaller and the site loads much faster. In v6.5, they will also bring lazy compilation of stories, which means the start-up/build of the site will be much faster.

Another notable feature is "interactive stories", which allows using Testing Library to interact with the story programmatically after it is first rendered and test that it behaves correctly.

- https://github.com/storybookjs/storybook/issues/15355
- https://storybook.js.org/blog/storybook-on-demand-architecture/
- https://storybook.js.org/blog/interactive-stories-beta/

Reply:
**Interactive stories** look really fun. I might write one for h5web to test it out